skipper.go

```diff
@@ -250,6 +250,11 @@ type Options struct {
 	// command this option is used when starting it with the -routes-file flag.)
 	WatchRoutesFile string
 
+	// RouteURLs are URLs pointing to route definitions, in eskip format, with change watching enabled.
+	// Multiple may be given comma separated. (For the skipper
+	// command this option is used when starting it with the -remote-routes-url flag.)
```

Comment:
The flag name was changed. Maybe we do not need the part in the brackets at all.

Reply:
It is also a slice now so `Multiple may be given comma separated.` is not relevant either

Reply:
Good catch.

Reply:
Fixed. Sorry for that. Simply missed that.